lib/rubocop/cop/style/redundant_line_continuation.rb

```diff
@@ -74,7 +74,7 @@ class RedundantLineContinuation < Base
         ALLOWED_STRING_TOKENS = %i[tSTRING tSTRING_CONTENT].freeze
         ARGUMENT_TYPES = %i[
           kDEF kFALSE kNIL kSELF kTRUE tCONSTANT tCVAR tFLOAT tGVAR tIDENTIFIER tINTEGER tIVAR
-          tLBRACK tLCURLY tLPAREN_ARG tSTRING tSTRING_BEG tSYMBOL tXSTRING_BEG
+          tLBRACK tLCURLY tLPAREN_ARG tSTRING tSTRING_BEG tSYMBOL tXSTRING_BEG tCOLON3
```

Comment:
Please move it to the appropriate position in alphabetical order, not to the end of the array, specifically between `kTRUE` and `tCONSTANT`.